frontend/packages/kubevirt-plugin/src/_style.scss

```diff
@@ -0,0 +1,2 @@
+@import '~kubevirt-web-ui-components/dist/sass/components';
```

Comment:
I think this is a good convention.

Each plugin can have all of its styling represented by a single `src/style.scss` file (this effectively fills the role of "stylesheet entry point").

In plugin's entry module, this file can be imported to load all the styling needed by that plugin.